arviz/plots/backends/bokeh/elpdplot.py

```diff
@@ -146,12 +147,14 @@ def _plot_atomic_elpd(
     ylabels_shown,
     plot_kwargs,
 ):
-    ax_.cross(
-        np.asarray(xdata),
-        np.asarray(ydata),
-        line_color=plot_kwargs.get("color", "black"),
-        size=plot_kwargs.get("s"),
+    marker = _validate_bokeh_marker(plot_kwargs.get("marker"))
+    marker_func = getattr(mk, marker)
+    sizes = np.ones(len(xdata)) * plot_kwargs.get("s")
```

Comment:
Maybe we could first try to read `"size"` and then `"s"`

    plot_kwargs.get("size", plot_kwargs.get("s"))

Reply:
I think this has to be handled before passing to the backend

Reply:
This is difference between mpl and bokeh

Reply:
Yes, I meant that if I remember correctly, the common code adds the "s" key to plot_kwargs

Reply:
It would still be the default

Reply:
Hey @ahartikainen @OriolAbril , This problem will be fixed after #1041 is fixed, which I'm currently working on. Should I add temporary check(for now) in main plot file?

Reply:
I think it can wait